src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/Razor/ExtractToCodeBehindCodeActionResolver.cs

```diff
@@ -34,7 +34,7 @@ internal class ExtractToCodeBehindCodeActionResolver : RazorCodeActionResolver
         private readonly DocumentResolver _documentResolver;
         private readonly FilePathNormalizer _filePathNormalizer;
 
-        private static readonly Range s_startOfDocumentRange = new Range(new Position(0, 0), new Position(0, 0));
+        private static readonly Range s_startOfDocumentRange = new Range { Start = new Position(0, 0), End = new Position(0, 0) };
```

Comment:
Since the new VS types are mutable lets not cache the "start of doc" range, lets new it up each time in the call site